api/TransformStream.json

```diff
@@ -87,6 +87,50 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "cancel": {
+          "__compat": {
+            "description": "`transformer.cancel` method",
+            "spec_url": "https://streams.spec.whatwg.org/#dom-transformer-cancel",
+            "support": {
+              "chrome": {
+                "version_added": false,
+                "impl_url": "https://crbug.com/40283531"
+              },
+              "chrome_android": "mirror",
+              "deno": {
+                "version_added": "1.38"
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false,
+                "impl_url": "https://bugzil.la/1856103"
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "nodejs": {
+                "version_added": "20.14.0"
+              },
```

Comment:
for node.js, seems it is also available in v21.7.0

Reply:
Right, it was *backported* from v21 to v20. I'll add v21.5.0 separately.